### ISSUE TITLE
how to change an already trained dense' activation? to generate sample

### ISSUE THREAD

**Issue by poppintiger** [keras-team/keras#9609]
say.I define a global Dense: 
 `dense=Dense(n_values,activation='softmax')`

I have  already trained the dense' activation;
when use the model generate sample,I want to change a little bit this  dense layer's activation function to use the same parameter to generate new output, so how can I do?To change an already trained dense' activation?

**Comment by poppintiger** [comment]
I use the custmized activation:
`
T=1
def Tsoftmax(x):
    numerator=K.exp(x/T)
    denominator =K.sum(K.exp(x/T),1)
    denominator =K.reshape(denominator,(-1,1)) 
    return numerator/denominator`
When I finish training the Dense, I  want to change T, but the result is always the same！
